package.sh

```diff
@@ -3,9 +3,13 @@
 readonly VE_LOCATION=/tmp/cexec_pkg_ve
 
 init_ve() {
+  local arch=$(uname -m)
   virtualenv -p /usr/bin/python $VE_LOCATION
   source $VE_LOCATION/bin/activate
-  pip install pyinstaller
+  # TODO: remove this check once pyinstaller 3.4 is released
+  if [ "$arch" != "aarch64" ]; then
+    pip install pyinstaller
```

Comment:
shouldn't this have a version?

Reply:
we use the latest version here. I will pin the verison to 3.3, if required

Reply:
- nothing should be installed without version pinning
- we'll upgrade when new versions come out but that cannot be implicit, so this needs to be fixed